### DISCUSSION TITLE
guide/run-js-script

### DISCUSSION THREAD

**Discussion by giscus** [become-cool/beshell#1] [Q&A]
# guide/run-js-script

运行 JavaScript 脚本文件 BeShell 应用的主要逻辑应该写在 JavaScript 文件中。因此需要 BeShell 完善了 ESP32 的文件系统支持 📄 。 将存储介质 (flash 或 sd卡) 上的文件分区挂载到 BeShell 的文件系统，然后就可以像在其他平台上一样运行 JavaScript 文件。 执行 BeShell:...

https://beshell.become.cool/guide/run-js-script.html

**Comment by qsbye** [comment]
和micropython比较有什么优势呢?
